### ISSUE TITLE
<area> with no href should allow roles

### ISSUE THREAD

**Issue by cyns** [w3c/html-aria#317]
`<a>` with no `href` takes any role, but `<area>` with no `href` takes no roles. These are similar and seem like they should be consistent. `<area role=button>` seems like it would be useful, as would checkbox and other widget roles.

**Comment by scottaohara** [comment]
I think it makes sense to adjust this, as Cynthia suggests / as we talked about in the ARIA WG meeting this past Thursday.  Being that `<area>` is specifically meant to be a child of a `<map>` I do not think it makes sense to allow _any_ role, as `<a>` without `href` does.  But checkboxes, radios, buttons... yeh, things like that could very well work.  

We can get this updated for the next version of the spec.

**Comment by stevefaulkner** [comment]
At the time I think that it was decided to not allow stuff on `<area>` because it does not have a visible UI by default, you  need to add `shape` and `coords` attributes to it then associate it with an `<img>`
We already provide a million ways for devs to shoot themselves and users in the foot, why provide another ;-)

**Comment by scottaohara** [comment]
closing this ticket in favor of 325 due to the w3cbot's incessant need to futz with the labels here.

**Comment by plehegar** [comment]
@michael-n-cooper  reminder: to remove needs-resolution, it needs to be removed on the horizontal side first, so I removed it in https://github.com/w3c/a11y-review/issues/74
